scripts/script.js

```diff
@@ -0,0 +1,23 @@
+/* Dog Info */
+function showInfo(name,breed,fee) {
+  let result = (name + ' is a ' + breed + ' and can be adopted for ' + fee + '.');
+  alert(result);
+}
+
+let total = 0;
+function addTotal(fee) {
+  let result = ('Your total is $' + (total += fee));
+  alert(result);
+}
+
+
+/* Checkout Form */
+function submitInfo() {
+  let entries = document.getElementsByTagName('input', 'option');
```

Comment:
I don't think you can specify multiple tag types in one function call. You should be doing one for input, and another for select.
Be careful with radio buttons, they are of type input but report using .checked instead of .value
You could check with something like: 
`if (entries[i].getAttribute("type") == 'radio') {`
`console.log(`Radio Button checked? ${entries[i].checked} `);`
`}`